README.md

```diff
@@ -1,211 +1,119 @@
+  - [libmagickwand-dev](http://docs.wand-py.org/en/0.2.4/guide/install.html)
+
+**Ubuntu or Debian:**
+You can install all the necessary packages using these commands:
+
+    # Install minio
     wget https://dl.minio.io/server/minio/release/linux-amd64/minio -O /usr/local/bin/minio
     chmod +x /usr/local/bin/minio
 
-    # install node PPA
+    # Install node PPA
     curl -sL https://deb.nodesource.com/setup_6.x | bash -
 
+    # Install packages
     apt-get install -y  python python-pip python-dev python-tk \
         postgresql-server-dev-all postgresql-contrib postgresql-client postgresql \
         ffmpeg nodejs libmagickwand-dev nginx redis-server wkhtmltopdf
 
-On Windows, you'll need to download and install Postgres and Redis manually.
-
-On Mac OS X, you can install the corresponding packages using Homebrew:
+**Mac OS X**
+You can install the corresponding packages using Homebrew:
 
     brew install  postgresql@9.6 redis node ffmpeg imagemagick@6 gs
     brew install minio/stable/minio
     brew link --force postgresql@9.6
     brew link --force imagemagick@6
 
+**Windows**
+Windows is no longer supported due to some of the required packages.
 
 
-##### Set up python dependencies through pipenv
+
+### 1.5. (Optional) Set up a virtual environment
+Run the following command (make sure you are under your project directory folder)
+
+    pipenv shell
+
+This will set up a virtual environment for you. To reactivate, run `pipenv shell`
+
+
+
+### 2. Set up python dependencies through pipenv
+Run the following commands to install the python dependencies listed in `Pipfile`
 
     pip install -U pipenv
     pipenv install
 
 
-The file `requirements_dev.txt` contains dependencies that will be helpful for
-development and required when using the `--settings=contentcuration.dev_settings`
-flag to run the server in development mode.
 
-Pytest and other test related dependencies are stored in `requirements_test.txt`.
+### 3. Set up pre-commit hooks
 
-##### Pre-Commit
-
-We use `pre-commit <http://pre-commit.com/>`_ to help ensure consistent, clean code. The pip package should already be installed from a prior setup step, but you need to install the git hooks using this command.
+We use [pre-commit](http://pre-commit.com/) to help ensure consistent, clean code. The pip package should already be installed from a prior setup step, but you need to install the git hooks using this command.
 
     pre-commit install
 
-##### Additional formatting tools
+
+#### Additional formatting tools
 
 In case you need help formatting your python code to meet pep8 standards, there are a couple tools out there.
-https://github.com/myint/autoflake for removing unused imports and unused variables.
-https://github.com/hhatto/autopep8 for fixing whitespace issues.
+    - [autoflake](https://github.com/myint/autoflake) for removing unused imports and unused variables
+    - [autopep8](https://github.com/hhatto/autopep8) for fixing whitespace issues.
```

Comment:
I think this is the sort of thing that doesn't belong in the README. It is one thing to say the project requires devs to use these tools, but as written this is general Python linting information, nothing specific to Studio.